grpc_opa/authorizer.go

```diff
@@ -252,6 +258,8 @@ func (a *DefaultAuthorizer) Evaluate(ctx context.Context, fullMethod string, grp
 	}
 
 	if !opaResp.Allow() {
+		forbidReason := opaResp.rbacForbiddenReason()
```

Comment:
Shouldn't we first check that an obligation was expected? This is going to be triggered and have no message for 100% of authz failures today.